kuka_rox_hardware_interface/src/rox_hardware_interface.cpp

```diff
@@ -112,14 +112,13 @@ export_command_interfaces()
       info_.joints[i].name,
       HW_IF_STIFFNESS,
       &hw_stiffness_[i]);
-  }
 
-  for (size_t i = 0; i < info_.joints.size(); i++) {
```

Comment:
@VX792 @Svastits In this situation is this the right way to correct the mistake, or it should be emplaced like POS, STIFFNESS, DAMPING, POS, STIFFNESS, etc?

Now it is placeing every POS param and after that the STIFFNESS and DAMPING param like STIFFNESS, DAMPING, STIFFNESS, DAMPING, etc.